java/test/jmri/implementation/PackageTest.java

```diff
@@ -47,15 +45,18 @@ public static Test suite() {
         suite.addTest(OffsetHighCvProgrammerFacadeTest.suite());
         suite.addTest(ResettingOffsetHighCvProgrammerFacadeTest.suite());
         suite.addTest(RouteTest.suite());
-        suite.addTest(SE8cSignalHeadTest.suite());
-        suite.addTest(SignalHeadSignalMastTest.suite());
         suite.addTest(SignalSpeedMapTest.suite());
         suite.addTest(SignalSystemFileCheckTest.suite());
         suite.addTest(SingleTurnoutSignalHeadTest.suite());
         suite.addTest(TwoIndexTcsProgrammerFacadeTest.suite());
         suite.addTest(BundleTest.suite());
+        // Converted to JUnit4:
         suite.addTest(new junit.framework.JUnit4TestAdapter(DccConsistTest.class));
+        suite.addTest(new junit.framework.JUnit4TestAdapter(DccSignalHeadTest.class));
```

Comment:
These should replace the existing tests in the order the existing tests were executed--there are a number of situations where test order matters (there shouldn't be, but that's a different issue).